.github/workflows/dockerhub.yml

```diff
@@ -47,6 +47,15 @@ on:
         description: "GitHub runner for workflow (Default: ubuntu-latest-16-cores)"
         type: "string"
         default: "ubuntu-latest-16-cores"
+      additional_build_args:
+        description: "Optional: Additional custom Docker build arguments e.g., UBUNTU_VERSION=jammy"
+        type: "string"
+        default: ""
+        required: false
+      override_tag:
```

Comment:
Can't we just use the suffix option for this and add jammy as the suffix

Reply:
I tried using our suffix inputs but I don't think those work for streets :

tag_name_suffix adds a suffix to the branch name, so we get tags like: develop-jammy, master-jammy, etc.

For both replace_suffix and remove_suffix: true will replace or remove the suffix,
but the result still keeps the branch name. For example: develop-jammy becomes develop after removing -jammy.

Reply:
Removed override tag input as we decided pushing with jammy and focal as suffix similar to other repo's